src/app/AnnotationsVariants/GRCh38/[snpid]/page.tsx

```diff
@@ -0,0 +1,183 @@
+"use client";
+
+import React, { useEffect, useMemo, useState } from "react";
+import { useParams } from "next/navigation";
+import {
+  Box,
+  Button,
+  Typography,
+  Divider,
+  Select,
+  MenuItem,
+  Breadcrumbs,
+  Link,
+  Grid,
+  CircularProgress,
+} from "@mui/material";
+import PeakIntersectionView from "./PeakIntersection";
+import MotifIntersectionView from "./MotifIntersection";
+import { useSNPData } from "../../hooks";
+import { ChainFile, loadChainFile } from "liftover";
+import NavigateNextIcon from "@mui/icons-material/NavigateNext";
+import NavigateBeforeIcon from "@mui/icons-material/NavigateBefore";
+import { inflate } from "pako";
+
+const AnnotationDetailPage = () => {
+  const [loadingChains, setLoadingChains] = useState(false);
+  const [chainFile, setChainFile] = useState<ChainFile | undefined>(undefined);
+
+  useEffect(() => {
+    setLoadingChains(true);
+    fetch(
```

Comment:
why is chainFile fetched at 2 places ? Can we fetch it once and use it wherever required ?